apps/codeforafrica/jest.setup.js

```diff
@@ -1 +1,9 @@
 module.exports = require("@commons-ui/testing-library/jest.setup");
+
```

Comment:
Having gone through the code and the problem, I kinda of agree with this [comment](https://github.com/jsdom/jsdom/issues/2524#issuecomment-902027138): The issue seem to be we're creating JSDOM inside Jest's jsdom environment and we should try avoid it. My quick suggestion that will make this setup unnecessary is in `lib/marked` file comment.